srcpkgs/ocean-sound-theme/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'sound-theme-ocean'
+pkgname=ocean-sound-theme
+version=6.1.4
+revision=1
+build_style=cmake
+hostmakedepends="extra-cmake-modules qt6-tools qt6-base"
+makedepends="qt6-tools-devel"
+short_desc="Ocean Sound Theme for Plasma"
+maintainer="Hunter Ellett <hunter2k1@disroot.org>"
+license="custom:multiple"
```

Comment:
```suggestion
license="CC-BY-SA-4.0"
```

(CI stuff is `CC0-1.0` but not used from us, CMake stuff is `BSD-2-Clause` but not installed, so not really important for the final package)